app/pages/patientdata/patientdata.js

```diff
@@ -952,7 +952,7 @@ export let PatientData = translate()(React.createClass({
       this.dataUtil.chartPrefs = this.state.chartPrefs[chartType];
 
       this.setState(state);
-      this.props.trackMetric(`web - default to ${chartType}`);
+      this.props.trackMetric(`web - default to ${chartType === 'bgLog' ? 'weekly' : chartType}`);
```

Comment:
I wonder if there's a mechanism for merging the two metrics via a query on the presentation end as opposed to shoehorning this in here.

Reply:
Yeah, worth looking into.  I didn't like doing it this way.  I'll merge this to develop in the meantime, and reach out to Brandon on the card to see if we can do this.